cache/cache.go

```diff
@@ -26,8 +26,11 @@ import (
 	clientv3 "go.etcd.io/etcd/client/v3"
 )
 
-// TODO: add gap-free replay for arbitrary startRevs and drop this guard.
-var ErrUnsupportedWatch = errors.New("cache: unsupported watch parameters")
+var (
+	// TODO: add gap-free replay for arbitrary startRevs and drop this guard.
+	ErrUnsupportedWatch = errors.New("cache: unsupported watch parameters")
+	ErrNotImplemented   = errors.New("cache.Get: not implemented yet")
```

Comment:
Don't know what's the difference between the two errors. Any reason to have Get and Watch specific errors?

Reply:
Sounds good. Please keep in mind there is a differences between unsupported and OutOfBounds (name could be improved) request for cache. A request for option that is not supoorted like WithPrevKV should return unsupported. A request for "/foo/a" key when cache monitors "/bar" prefix is OutOfBounds.